manim/mobject/graph.py

```diff
@@ -23,91 +24,296 @@
 from manim.mobject.opengl.opengl_mobject import OpenGLMobject
 from manim.mobject.text.tex_mobject import MathTex
 from manim.mobject.types.vectorized_mobject import VMobject
+from manim.typing import Point3D
 from manim.utils.color import BLACK
 
+NxGraph: TypeAlias = Union[nx.classes.graph.Graph, nx.classes.digraph.DiGraph]
```

Comment:
```suggestion
NxGraph: TypeAlias = nx.classes.graph.Graph | nx.classes.digraph.DiGraph
```
I would also prefer this to be under an `if TYPE_CHECKING` condition. Note that you might need to add a `from __future__ import annotations`